ozaria/site/components/cutscene/common/BaseVideo.vue

```diff
@@ -9,39 +9,67 @@ export default {
       type: String,
       required: false
     },
+
     videoSrc: {
       type: String,
       required: false
     },
+
+    isSoundOn: {
+      type: Boolean,
+      default: true
+    },
+
     captions: {
       type: Array,
       default: ()=>([]),
       required: false
     }
   },
-  mounted: function() {
+
+  data: () => ({
+    vimeoPlayer: null
+  }),
+
+  async mounted () {
     if (!(this.vimeoId || this.videoSrc)) {
       throw new Error('You must pass in a "vimeoId" or a "videoSrc"')
     }
 
     if (this.vimeoId) {
-      const player = new VimeoPlayer('vimeo-player')
-      player.ready().then(() => {
-        player.play()
-      })
+      const player = this.vimeoPlayer = new VimeoPlayer(this.$refs['vimeo-player'])
+      await player.ready()
+      try {
+        await player.setVolume(this.isSoundOn ? 1 : 0)
+        await player.play()
+      } catch (e) {
+        console.warn(`Wasn't able to auto play video.`)
+      }
       // TODO: Instead of emitting completed, requires end screen UI.
       //        Currently a stop gap to provide Intro support.
       player.on('ended', () => this.$emit('completed'))
     } else if (this.videoSrc) {
       new Plyr(this.$refs['player'], { captions: {active: true } })
     }
+  },
+
+  watch: {
+    isSoundOn(handleSoundOn) {
+      if (!this.vimeoPlayer) {
+        return
+      }
+
+      // TODO: This can sometimes pause the video when turning on the volume.
+      this.vimeoPlayer.setVolume(handleSoundOn ? 1 : 0)
```

Comment:
Small suggestion: move this into a method so we have a single place that handles volume setting.  Then call that method here and above on line 43